package.json

```diff
@@ -4,17 +4,23 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
-    "dev": "nodemon app.js",
+    "start": "cross-env NODE_ENV=development node app.js",
+    "dev": "cross-env NODE_ENV=development nodemon app.js",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Suggest adding scripts like `db:migrate` or `db:seed` to specify the manual operation.

Reply:
added "migrate" and "seed" in scripts
and updated it in README.md as well